code/modules/organs/organ_external.dm

```diff
@@ -573,11 +573,13 @@ This function completely restores a damaged organ to perfect condition.
 
 //Determines if we even need to process this organ.
 /obj/item/organ/external/proc/need_process()
```

Comment:
Can we just have the `BP_IS_ROBOTIC` check first? I think they don't ever need to process, at all? And if we do it that early then we skip some stuff in `get_pain()` because `get_pain()` doesn't work on robotic organs anyway.

Reply:
handled